### PR TITLE
13504 correction document

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#13504

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/report-templates/correction.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+[[macros.html]]
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <title>Register Correction Application</title>
```
Comment (argush3) [inline legal-api/report-templates/correction.html:5]: do we want to remove the word "Register" or use a different title altogether?

File: legal-api/report-templates/correction.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+   <body>
+      <div class="header">
+         <table class="header-table" role="presentation">
+            <tr class="no-page-break">
+               <td>
+                  [[logo.html]]
+               </td>
+               <td>
+                  <div class="report-type">
+                     REGISTER CORRECTION APPLICATION
```
Comment (argush3) [inline legal-api/report-templates/correction.html:19]: same comment as above regarding title.

File: legal-api/report-templates/correction.html (hunk 1 above)
Comment (thorwolpert) [reply legal-api/report-templates/correction.html:5]: Agreed, I think it should wither be _Registry Correction_ or just _Correction_

**Review by thorwolpert — COMMENTED** [review]
Just a discussion point on the Correction filing output's title.

Do we have rendered output to review?
I do see the correction portions removed from the various segments, and that looks correct, but a rendered version might be easier to approve as well.

**Comment by vysakh-menon-aot** [comment]
image

File: legal-api/report-templates/correction.html (hunk 1 above)
Comment (vysakh-menon-aot) [reply legal-api/report-templates/correction.html:5]: I will verify this with Mihai

Comment (vysakh-menon-aot) [reply legal-api/report-templates/correction.html:5]: Confirmed with Mihai. This is what business want, we have the same title in Firm Correction.

**Review by argush3 — APPROVED** [review]

**Review by ChrisGabel23 — APPROVED** [review]

**Review by thorwolpert — APPROVED** [review]
Thanks for including a sample output.

**Pull request merged**